Packs/BitcoinAbuse/Integrations/BitcoinAbuse/BitcoinAbuse.py

```diff
@@ -7,70 +7,75 @@
 
 ''' CONSTANTS '''
 SERVER_URL = 'https://www.bitcoinabuse.com/api/'
-FEED_ENDPOINT_SUFFIX = 'download/'
-FEED_ENDPOINT_DAILY_SUFFIX = 'download/1d'
-REPORT_ADDRESS_SUFFIX = 'reports/create'
-abuse_type_name_to_id: Dict[str, int] = {
-    'ransomware': 1,
-    'darknet market': 2,
-    'bitcoin tumbler': 3,
-    'blackmail scam': 4,
-    'sextortio': 5,
-    'other': 99
-}
-abuse_type_id_to_name: Dict[str, str] = {
+ABUSE_TYPE_ID_TO_NAME: Dict[str, str] = {
     '1': 'ransomware',
     '2': 'darknet market',
     '3': 'bitcoin tumbler',
     '4': 'blackmail scam',
     '5': 'sextortio',
     '99': 'other'
 }
-first_fetch_interval_to_url_suffix: Dict[str, str] = {
-    'Forever': 'forever',
-    '30 Days': '30d'
-}
 
 READER_CONFIG = {
     'fieldnames': ['id', 'address', 'abuse_type_id', 'abuse_type_other', 'abuser',
                    'description', 'from_country', 'from_country_code', 'created_at'],
     'skip_first_line': True,
     'indicator_type': 'Cryptocurrency Address',
     'mapping': {
-        'Value': ('address', None, 'bitcoin-{}'),
+        'Value': ('address', None, 'bitcoin:{}'),
         'rawaddress': 'address',
         'countryname': 'from_country',
         'creationdate': 'created_at',
         'description': 'description',
-        'abusetype': ('abuse_type_id', lambda abuse_type_id: abuse_type_id_to_name.get(abuse_type_id))
+        'abusetype': ('abuse_type_id', lambda abuse_type_id: ABUSE_TYPE_ID_TO_NAME.get(abuse_type_id))
     }
 }
 
 
 class BitcoinAbuseClient(BaseClient):
+    first_fetch_interval_to_url_suffix: Dict[str, str] = {
+        'Forever': 'forever',
+        '30 Days': '30d'
+    }
+
+    abuse_type_name_to_id = {
+        'ransomware': 1,
+        'darknet market': 2,
+        'bitcoin tumbler': 3,
+        'blackmail scam': 4,
+        'sextortio': 5,
+        'other': 99
+    }
```

Comment:
Why have we decided to put this in the client after all? the only place this is used is outside the client.
Also capitalize.

Reply:
You're right, capitalised and moved it outside